File: ExpressJS Overview - Calculator/models/Calculator.js
function Calculator(leftOperand, operator, rightOperand) {
    this.leftOperand = leftOperand;
    this.operator = operator;
    this.rightOperand = rightOperand;

    this.calculate = function () {
        if (Number.isNaN(leftOperand) || Number.isNaN(rightOperand)) {
            return 'NaN';
        }

        let result = 0;

        switch (this.operator) {
            case '+': {
                result = this.leftOperand + this.rightOperand;
                break;
            }
            case '-': {
                result = this.leftOperand - this.rightOperand;
                break;
            }
            case '*': {
                result = this.leftOperand * this.rightOperand;
                break;
            }
            case '/': {
                if (this.rightOperand == 0) {
                    return 'Division By 0';
                }
                result = this.leftOperand / this.rightOperand;
                break;
            }
            case '%':
            {
                result = this.leftOperand % this.rightOperand;
                break;
            }
            case '^':
            {
                result = this.leftOperand ** this.rightOperand;
                break;
            }
        }

        return result;
    }
}

module.exports = Calculator;

File: CS - ASP.NET MVC Overview/Calculator/Models/Calculator.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace Calculator_CSharp.Models
{
    public class Calculator
    {
        public double LeftOperand { get; set; }
        public double RightOperand { get; set; }

        public string Operator { get; set; }

        public double Result { get; set; }

        public Calculator()
        {
            this.Result = 0;
        }
    }
}